frontend/src/pages/SubProjects/reducer.js

```diff
@@ -71,12 +78,20 @@ const defaultState = fromJS({
   displayNameForPermissions: "",
   showProjectAssignees: false,
   projectAssignee: "",
-  dialogTitle: strings.subproject.subproject_add_title
+  dialogTitle: strings.subproject.subproject_add_title,
+  searchTerm: "",
```

Comment:
couldn't searchTerm, highlightingRegex reused from project search/ currently overview? Maybe we should put the search bar in an own container to give it an own reducer. So the same state values searchTerm, searchTerms[] and highlightlingRegex can be used for all searchbars implemented. The reason why it's possible is because only one searchbar can be active/displayed at once.

Reply:
what's the solution to this comment?

Reply:
if searchTerm, searchBarDisplayed, highlightingRegex are just once for searchbar and re-used for project- and subproject-level, it would work normally BUT this results in few flexibility, since only one searchterm can exist. When implementing searchbar on history-level, the searchterm and searchresults on subproject-level are overwritten.

Reply:
ok I agree, the states searchTerm, searchBarDisplayed, highlightingRegex should be stored individually for all searchbars, but every individual searchbar should use the Common searchbar component including Navbar.

Reply:
I see ... since the searchbar is a common component move it to the  Common folder